apps/dashboard/app/models/script.rb

```diff
@@ -45,12 +48,15 @@ def next_id(project_dir)
     end
   end
 
+  attr_reader :project_dir, :smart_attributes
+
   def initialize(opts = {})
     opts = opts.to_h.with_indifferent_access
 
     @project_dir = opts[:project_dir] || raise(StandardError, 'You must set the project directory')
     @id = opts[:id]
     @title = opts[:title].to_s
+    @smart_attributes = opts[:smart_attributes] || {}
```

Comment:
So here we're going to read an array from `opts[:form]`.

Remember what the YAML looks like:

```
title: 'some title'
form:
  - auto_accounts

Reply:
What is that `:form` key? I figured these would be key/values, but i can make it an array, i jsut don't quite get how that data struct works with all this.

Reply:
as in it is getting an array, and depending on order the key values are matched up?

Reply:
It follows the same pattern as a `batch_connect` application.

Reply:
I.e., it'll read/behave just like a batch_connect's `form.yml`  That's the functionality we're going for, generating HTML forms from YAML almost just like `batch_connect` applications do.